gulp/ui-lib.js

```diff
@@ -70,9 +70,13 @@ function copyJsTask() {
 copyJsTask.displayName = taskNamePrefix + 'js';
 copyJsTask.description = 'Copies JS files.';
 
+function copyImgTask() {
```

Comment:
the task should be renamed so that it's clear that it will copy images that will go into the UI library. (see following comment)

Reply:
I've removed this, it's now handled by patternlab